tests/_support/Page/Acceptance/Administrator/ControlPanelPage.php

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace Page\Acceptance\Administrator;
+
+use Page\Acceptance\Administrator\AdminPage;
+
+class ControlPanelPage extends AdminPage
+{
+    public static $url = "/administrator/index.php";
+
+    public static $pageTitle = 'Control Panel';
+
+    public static $pageTitleContext = ['class' => 'page-title'];
```

Comment:
I mean that with your IDE, right now you can see the value of `ControlPanelPage::pageTitleContext` with just Ctrl+click over the parameter. But if we do Ctrl+click at `Article::_('articleTitleField')` your IDE will navigate to the `public function _(...)` so you wont be able to easily navigate to the value.